src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -819,6 +819,7 @@ TTL747266 = 747266 : quadruple portes XNOR 2 entrées
 TTL7474 = 7474 : deux bascules D prépositionnables et réinitialisables
 TTL7485 = 7485 : comparateur de magnitude 4 bits
 TTL7486 = 7486 : quadruple portes OU exclusif (XOR) à 2 entrées
+# ==> TTL7487 =
```

Comment:
French translation: "7487 : 4 bits Vrai/complément, éléments zéro/un"

Reply:
Added in commit